spaceship/lib/spaceship/connect_api/models/app.rb

```diff
@@ -244,6 +244,14 @@ def get_app_store_versions(filter: {}, includes: Spaceship::ConnectAPI::AppStore
         return resps.flat_map(&:to_models)
       end
 
+      def get_app_store_versions_count(filter: {}, limit: nil)
+        return Spaceship::ConnectAPI.get_app_store_versions(
+          app_id: id,
+          filter: filter,
+          includes: Spaceship::ConnectAPI::AppStoreVersion::ESSENTIAL_INCLUDES,
+          limit: limit
+        ).paged_resources_count
+      end
```

Comment:
@pNre This is really great find! One quick question. Instead of creating a method for this, can we instead just not call `.all_pages` if `limit` isn't `nil` in `get_app_store_versions`? I think that might make things a little cleaner and make that method behave as expect with limit.

Reply:
I think it depends on how limit works and how long pages are.
For example, if `limit = 30` and pages have at most `20` items, don't I still need to paginate to get the last 10 items?

Reply:
Hmm... good point! What about my next suggestion above? 😇  This seems pretty specific for `deliver` so I think I'd like avoid adding a method for count in `Spaceship::ConnectAPI::App` 🤔

Reply:
@joshdholtz change implemented

Reply:
Thank you so much! I'll get this merged and out the door later today or tomorrow ❤️